portfolio/src/main/webapp/index.html

```diff
@@ -152,7 +151,8 @@ <h3>My Portfolio Site </h3>
     <div class="blankspace"></div> <!-- necessary to allow for scrolling to blog section" -->
 
     <footer>
-        <button onclick="msg()" style="float:right; margin-bottom:20px;"><i class="fa fa-commenting-o fa-4x"aria-hidden="true"></i></button>
+        <
+        <button onclick="msg()" style="float:right; margin-bottom:20px;" class="fa fa-commenting-o fa-4x"aria-hidden="true"></button>
```

Comment:
`aria` attributes are for accessibility.  Why are you marking this as hidden for accessibility purposes? (If you're not putting accessibility markers on anything else, I wouldn't bother here I think).

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

Reply:
I got the icon snippet from font awesome directly, will remove all instances.